Dockerfile

```diff
@@ -1,14 +1,21 @@
-FROM registry.access.redhat.com/ubi8/python-39
+FROM quay.io/centos/centos:stream9-development
 
 ARG USER_ID=${USER_ID:-1001}
+ARG APP_DIR=${APP_DIR:-/app}
 ARG DEVEL_COLLECTION_LIBRARY=0
-WORKDIR $HOME
 
 USER 0
-RUN dnf install -y java-17-openjdk-devel
+RUN useradd -u $USER_ID -d $APP_DIR appuser
+WORKDIR $APP_DIR
+COPY . $WORKDIR
```

Comment:
Oh I didn't notice that we are not using a requirements file for the app dependencies. Create a new file for it would need more changes out of the scope of your PR and I want to migrate the project to poetry at some point. So we can do this improvement later.